clients/ethereumjs/Dockerfile

```diff
@@ -1,15 +1,86 @@
-### Build EthereumJS From Git:
+### Build Ethereumjs Locally:
+### Requires a copy of ethereumjs-monorepo/ -> hive/clients/ethereumjs/
 
-FROM node:18-alpine as build
+FROM node:18-alpine
 
-ARG github=ethereumjs/ethereumjs-monorepo
-ARG tag=master
+RUN apk update && apk add --no-cache bash git jq curl python3 gcc make g++ \
+    && rm -rf /var/cache/apk/*
 
-RUN echo "Cloning: $github - $tag" \
-    && apk update && apk add --no-cache bash git jq curl python3 gcc make g++ \
-    && rm -rf /var/cache/apk/* \
-    && git clone --depth 1 -b $tag https://github.com/$github \
-    && cd ethereumjs-monorepo && npm i
+RUN npm i -g ts-node
+
+# Default local client path: clients/ethereumjs/<ethereumjs-monorepo>
+ARG local_path=ethereumjs-monorepo
+
+
+COPY ${local_path}/package-lock.json ethereumjs-monorepo/package-lock.json
+COPY ${local_path}/package.json ethereumjs-monorepo/package.json
+
+COPY ${local_path}/.editorconfig ethereumjs-monorepo/.editorconfig
```

Comment:
are these really needed? may be we an just skip these

Reply:
I have naively copied those. These small copies take almost 0 time. (However, it might create a new base image, so I am not sure if it takes a lot of disk space). These can likely be removed.